PostgreSQL/node.prepare.json

```diff
@@ -0,0 +1,34 @@
+{
+	"name": "PostgreSQL Prepare Statement",
+	"fileName": "node.prepare.js",
+	"inputSchema": [
+		{
+			"key": "query",
+			"type": "String"
+		},
+		{
+			"key": "values",
+			"type": "Array"
+		}
+	],
+	"outputSchema": [
```

Comment:
Output schema doesn't match what is being returned. This says that the output would be `{results: <...>}`

Reply:
Changed this. Now we return rowCount and rows both. This works for any query.